README.md

```diff
@@ -863,12 +856,12 @@ Since the Web SDK renders in-app messages in an iframe element on your website i
 
 But there are few features which the SDK adds so that you can customize how you'd like links to behave:
 
-First, the [`handleLinks` option](#getInAppMessages) within the `getMessages` call will allow you to either open all links in a new tab, the same tab, or ensure that external links open in a new tab, while internal ones keep the experience within the same tab. So for example, this code:
+First, the [`handleLinks` option](#getInAppMessages) within the `getInAppMessages` call will allow you to either open all links in a new tab, the same tab, or ensure that external links open in a new tab, while internal ones keep the experience within the same tab. So for example, this code:
```

Comment:
```suggestion
First, the `handleLinks` option provided by [`getInAppMessages`](#getInAppMessages) allows you to specify how the SDK opens links: in the current tab, in a new tab, or a combination (external links in a new tab, internal links in the current tab). For example, this code:
```

@mprew97 / @pauljung14  It isn't clear to me based on the text here how these options work — if links open in a new tab, do they open in the _same_ new tab? Or multiple new tabs?

Reply:
i would have to mess with the `open-all-new-tab` option to clarify. For the sake of this PR, we should just roll with this change and I'll cut a separate issue to look into that behavior.